templates/default/flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  # Add project name.
+  description = "your_project";
+
+  nixConfig = {
+    bash-prompt-prefix = "(rules_ll) ";
+    bash-prompt = ''\[\033]0;\u@\h:\w\007\]\[\033[01;32m\]\u@\h\[\033[01;34m\] \w \$\[\033[00m\]'';
+    bash-prompt-suffix = " ";
+  };
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    rules_ll.url = "github:eomii/rules_ll";
+  };
+
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    , rules_ll
+    , ...
+    }:
+    flake-utils.lib.eachSystem [ "x86_64-linux" ] (system:
+    let
+      pkgs = import nixpkgs {
+        inherit system;
+      };
+    in
+    {
+      devShells = {
+        default = rules_ll.lib.${system}.llShell {
+          # Set to true for NVPTX support, make sure to read the CUDA license.
+          unfree = false;
+          # Add packages.
+          packages = [
+            # pkgs.openssl
+          ];
+          # Set environment variables.
+          env = {
+            # LL_CFLAGS = " ";
+            # LL_LDFLAGS = " ";
+          };
```

Comment:
This should probably have either the same layout as in https://ll.eomii.org/guides/external_dependencies/#example or just reference the guide. (Or both?)